pages/linux/genfstab.md

```diff
@@ -0,0 +1,20 @@
+# genfstab
+
+> Generate output suitable for addition to an fstab file.
+> More information: <https://man.archlinux.org/man/extra/arch-install-scripts/genfstab.8>.
+
+- Generate the output based on a label:
+
+`genfstab -L {{path/to/mount_point}}`
+
+- Display `fstab` output using partitions' UUID:
+
+`genfstab -U {{path/to/mount_point}}`
+
+- Generate an fstab file, requires being a superuser:
+
+`genfstab -U {{/mnt}} >> {{/mnt/etc/fstab}}`
```

Comment:
How about this:
```suggestion
`genfstab -U {{path/to/mount_point}} >> {{/mnt/etc/fstab}}`
```

Reply:
I would rather change the description to: 

- Most usual way to generate an fstab file, requires being a superuser:

And left the code the way it is. What do you think?

Reply:
1. IMO, changing the description isn't necessary. I think it's good already 👍🏻 
2. `/mnt` is inconsistent it seems with the rest, that may be intentional (I'm no Arch user :smile:)

Reply:
Actually, I think it would be better to make this example more consistent with the last one, which would solve the superuser problem, by using `sudo tee`:
```suggestion
`genfstab -U {{path/to/mount_point}} | sudo tee -a {{/mnt/etc/fstab}}`
```

Reply:
If you're sure that `/mnt` is better, even though it's inconsistent, let it be that way. I trust you know better :)

Reply:
@bl-ue in this example @peterbabic tried to explain the most common use case for this command as a step in the Arch Linux installation https://wiki.archlinux.org/index.php/installation_guide#Fstab. Nobody uses `sudo tee` 😄 because the installer is a live ISO and you run it with `root` user, command by command.

Reply:
Look, we can even discard the first and the second example. The third and the fourth are the only ones that I have personally ever needed and will serve for most people.

Reply:
Yeah, but the first 2 examples are useful to explain how the command works and some arguments.

Reply:
Yes exactly, that is why I have added there. I was following the contribution guidelines.

Reply:
Okay guys, great!  @navarroaxel cool that you're an Arch Linux guy.  That's good to know.

> Nobody uses `sudo tee` 😄 because the installer is a live ISO and you run it with root user, command by command.

😄😄😄😄😄